charts/kube-prometheus-stack/values.yaml

```diff
@@ -2234,13 +2234,13 @@ prometheusOperator:
     enableProbe: false
 
     # resource config for prometheusConfigReloader
-    resources:
-      requests:
-        cpu: 200m
-        memory: 50Mi
-      limits:
-        cpu: 200m
-        memory: 50Mi
+    resources: {}
```

Comment:
Is there a reason you cannot override the value when you deploy the chart? This value works for most of the users

Reply:
I see that a lot of charts do not set resources with default values.
And i would like to set resources without `cpu` limit.

Reply:
In this particular case, setting a resource to "0" will disable that limit/request. If support for setting `resources` empty is desired, further changes will be required in the operator's deployment template.